### PR TITLE
blog: health-checking-grpc

### PR DESCRIPTION
**Pull request by ahmetb**
date in the markdown and file/dir names will be updated later

### CODE REVIEW

File: content/en/blog/_posts/2018-09-30-health-checking-grpc.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+[![options for health checking grpc on kubernetes today](/images/blog/2019-09-30-health-checking-grpc/options.png)](/images/blog/2019-09-30-health-checking-grpc/options.png)
+
+
+1.  **httpGet probe:** Cannot be natively used with gRPC. You need to refactor
+    your app to serve both gRPC and HTTP/1.1 protocols (on different port
+    numbers).
+2.  **tcpSocket probe:** Opening a socket to gRPC server is not meaningful,
+    since it cannot read the response body.
+3.  **exec probe:** This invokes a program in container's ecosystem
```
Comment (neolit123) [inline content/en/blog/_posts/2018-09-30-health-checking-grpc.md]: `in container's` -> `in a container's` ?

File: content/en/blog/_posts/2018-09-30-health-checking-grpc.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    periodically. In gRPC case, this means you implement a health RPC yourself,
```
Comment (neolit123) [inline content/en/blog/_posts/2018-09-30-health-checking-grpc.md]: `In gRPC case` -> `In the case of gRPC` ?

File: content/en/blog/_posts/2018-09-30-health-checking-grpc.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    then write and ship a client tool with your container.
+
+Can we do better? Absolutely.
+
+## Introducing grpc-health-probe
+
+To standardize the "exec probe" approach mentioned above, we need:
+
+- a **standard** health check "protocol" that can be implemented in any gRPC
+  server easily,
```
Comment (neolit123) [inline content/en/blog/_posts/2018-09-30-health-checking-grpc.md]: `easily,` -> `easily.`

**Comment by ahmetb** [comment]
Thanks for the review. Addressed feedback: [[staging url]](https://deploy-preview-10393--kubernetes-io-master-staging.netlify.com/blog/2018/09/30/health-checking-grpc/)

**Pull request merged**